### ISSUE TITLE
URLlib3 errors when importing

### ISSUE THREAD

**Issue by ianhyzy** [sampsyo/beets#1275]
When importing, beets will continuously print:

```
C:\Python27\lib\site-packages\requests\packages\urllib3\connectionpool.py:734:

InsecureRequestWarning: Unverified HTTPS request is being made.

Adding certificate verification is strongly advised.

See: https://urllib3.readthedocs.org/en/latest/security.html
```

Here's my config:
```
directory: D:\SyncMusic
library: ~/musiclibrary.blb
plugins: fetchart lyrics lastgenre
import:
    move: yes
fetchart:
    cautious: true
    cover_names: front
```

**Comment by sampsyo** [comment]
Thanks for reporting. This was fixed in #1204.

**Comment by ianhyzy** [comment]
Thanks, sorry I didn't see that when I searched.
